sample/pom.xml

```diff
@@ -0,0 +1,74 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+	xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/maven-v4_0_0.xsd">
+	<modelVersion>4.0.0</modelVersion>
+	<groupId>ro.nttdata.ligaaclabs</groupId>
+	<artifactId>sample</artifactId>
```

Comment:
Ideal ar fi sa redenumim  artifact-ul in "dojo-management" sau ceva similar - e ok si sample in prima faza